pom.xml

```diff
@@ -46,7 +46,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.12</version>
+      <version>4.13</version>
```

Comment:
I think, you need to rebase/merge because this change has already been made in a previous PR

Reply:
Not sure why it's showing a diff

Reply:
I changed the version on my local files too, this should be resolved in the merge.